src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+}
+
+.stars--2 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+}
+
+.stars--3 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row;
+  height: 16px;
```

Comment:
Check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)